### ISSUE TITLE
add TorrentGalaxy error Query successful, but no results were returned from your indexer. This may be an issue with the indexer, your indexer category settings, or other indexer settings such as search freeleech only

### ISSUE THREAD

**Issue by tifo71** [Prowlarr/Indexers#459]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

there's an error when adding TorrentGalaxy in prowlarr here's a screenshot

![image](https://github.com/Prowlarr/Prowlarr/assets/130233722/6976eeae-9084-4ab0-8310-6738b52a3c44)

### Expected Behavior

adding torrentgalaxy works as before

### Steps To Reproduce

to reproduce the error, you need to add torrentgalaxy and when you test it, you get the error

### Environment

```markdown
- OS: windows 11 (opermediavault in virtual machine)
- Prowlarr: 1.18.0.4543
- Docker Install: yes
- Using Reverse Proxy: no
- Browser: opera one 110.0.5130.49
```

### What branch are you running?

Master

### Trace Logs?

[prowlarr.trace.txt](https://github.com/user-attachments/files/15534509/prowlarr.trace.txt)

### Trace Logs have been provided as applicable. Reports may be closed if the required logs are not provided.

- [X] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by tifo71** [comment]
sorry for the duplication. I couldn't find my post.

**Comment by tifo71** [comment]
> Same issue on [Jackett](https://github.com/Jackett/Jackett/issues/15358#issuecomment-2144183831), it seems they added an another checkpoint which makes it dead at this point.

on jackett for torrentgalaxy it work

**Comment by tifo71** [comment]
![image](https://github.com/Prowlarr/Indexers/assets/130233722/3b6078b3-53c2-4344-b88c-927d531f6ade)

it's a new checkpoint. before it was galaxyfance and now galaxy checkpoint

**Comment by AlexandruNegura** [comment]
Is this getting looked at?

**Comment by tifo71** [comment]
> Is this getting looked at?

I don't think so. Someone told me on discord that TorrentGalaxy no longer allowed you to automate cookies or something. In the end, it won't be possible to integrate the new security system.

**Comment by bakerboy448** [comment]
There's nothing to look at.
TG has decided to decrease their TTL to the point automation is not usable.

**Comment by bakerboy448** [comment]
There are still some issues being worked out with their checkpoint solution, and they don't want to throw the combined public-tracker-using userbases of Jackett and Prowlarr (maybe DuckieTV too) at it just yet. There is a potential solution in mind, but we're waiting for TGx to give us the green light.

Keeping it marked as down just to remind me to check for updates.

_Originally posted by @ilike2burnthing in https://github.com/Jackett/Jackett/issues/15358#issuecomment-2156095134_

**Comment by tifo71** [comment]
> There are still some issues being worked out with their checkpoint solution, and they don't want to throw the combined public-tracker-using userbases of Jackett and Prowlarr (maybe DuckieTV too) at it just yet. There is a potential solution in mind, but we're waiting for TGx to give us the green light.
> 
> Keeping it marked as down just to remind me to check for updates.

Great thanks
